src/archiefvernietigingscomponent/emails/models.py

```diff
@@ -0,0 +1,147 @@
+import copy
+import logging
+import re
+
+from django.conf import settings
+from django.core.mail import EmailMessage
+from django.db import models
+from django.urls import reverse
+from django.utils.translation import ugettext_lazy as _
+
+from solo.models import SingletonModel
+
+from archiefvernietigingscomponent.accounts.models import User
+from archiefvernietigingscomponent.emails.constants import (
+    EmailPreferenceChoices,
+    EmailTypeChoices,
+)
+
+logger = logging.getLogger(__name__)
+
+
+USER_TEMPLATE_ELEMENT = "{{ user }}"
+MUNICIPALITY_TEMPLATE_ELEMENT = "{{ municipality }}"
+DL_TEMPLATE_ELEMENT = "{{ list }}"
+LINK_DL_TEMPLATE_ELEMENT = "{{ link_list }}"
+LINK_REPORT_TEMPLATE_ELEMENT = "{{ link_report }}"
+
+EMAIL_TEMPLATE_ELEMENTS = (
+    USER_TEMPLATE_ELEMENT,
+    MUNICIPALITY_TEMPLATE_ELEMENT,
+    DL_TEMPLATE_ELEMENT,
+    LINK_DL_TEMPLATE_ELEMENT,
+    LINK_REPORT_TEMPLATE_ELEMENT,
+)
+
+
+class EmailConfig(SingletonModel):
+    municipality = models.CharField(
+        _("municipality"),
+        max_length=200,
+        help_text=_("The municipality on behalf of which the emails are sent."),
+    )
+
+    class Meta:
+        verbose_name = _("email configuration")
+
+
+class AutomaticEmail(models.Model):
+    type = models.CharField(
+        _("type"),
+        max_length=200,
+        help_text=_("The type of email"),
+        choices=EmailTypeChoices,
+        unique=True,
+    )
+    body = models.TextField(_("body"), help_text=_("The text sent in the email body"))
+    subject = models.CharField(
+        _("subject"), max_length=200, help_text=_("The text sent in the email subject")
+    )
+
+    class Meta:
+        verbose_name = _("Automatic email")
+        verbose_name_plural = _("Automatic emails")
+
+    def __str__(self):
+        return f"Automatic email ({self.type})"
+
+    def send(
+        self, recipient, destruction_list, report=None,
+    ):
+        """
+        :param recipient: type User
+        :param destruction_list: type DestructionList:
+        :param report: type DestructionReport (optional)
+        :rtype: None
+        """
+        email = EmailMessage(
+            subject=self.subject,
+            body=self.compose_body(recipient, destruction_list, report),
+            from_email=settings.DEFAULT_FROM_EMAIL,
```

Comment:
Let's make `from_email` configuratble in an existing solo config so we can configure the email address in the admin. Default can be  `archiefvernietigingscomponent@example.com`.